auth_token/contrib/common/views.py

```diff
@@ -13,3 +15,7 @@ class LoginView(TokenLoginView):
 
 class LogoutView(TokenLogoutView):
     pass
+
+
+class LoginCodeVerificationView(_LoginCodeVerificationView):
```

Comment:
I couldn't figure out better name, so each view is named the same "LoginCodeVerificationView". Is it ok or should I change it?